test/FixCode/fixits-apply.swift.result

```diff
@@ -33,7 +33,7 @@ func supported() -> MyMask {
 
 func foo() -> Int {
   do {
-  } catch let err {
```

Comment:
I don't understand why you made this test suite change, it doesn't seem necessary.  If it is necessary, then there is something else going wrong.  We want to still allow "catch let err".